build/lifecycle.go

```diff
@@ -134,3 +163,20 @@ func randString(n int) string {
 	}
 	return string(b)
 }
+
+func (l *Lifecycle) hasVersion020OrLater() bool {
```

Comment:
I wonder if we need to be this strict about the version format, it would be easy to do `strings.Compare` instead (semvers are lexicographically increasing) but i'm open to parsing the semver if we feel it is better.

Reply:
True, though it compares incorrectly for things like `0.2.0-rc.1` vs `0.2.0`. That said, for the purpose of determining whether 0.2.0 features are supported, both `0.2.0-rc.1` and `0.2.0` satisfy `>= 0.2.0`. We'll make the change.